Apps/Inc/Tasks.h

```diff
@@ -140,24 +135,10 @@ typedef enum{
     OS_MAIN_LOC = 0x200,
     OS_CANDRIVER_LOC = 0x400,
     OS_MOTOR_CONNECTION_LOC = 0x800,
-    OS_DISPLAY_LOC = 0x1000
+    OS_DISPLAY_LOC = 0x1000,
+    OS_TASKS_LOC = 0x2000,
```

Comment:
Could I get an explanation of the task loc?

Reply:
Original intent was to be able to see where the OS error was coming from in the case of an OS fault. So we would call the assertion function and pass in the location of the error (which almost directly corresponds to a file). It's scheduled for removal in #333 since we now have the previous tasks array that @Cam0Cow implemented.

Reply:
I added OS_TASKS_LOC to use in assertOSError calls that happen in Tasks.c, specifically the assertTaskError function.